packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -106,19 +91,17 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   // Start the react-native instance, which will create a JavaScript runtime and load the applications bundle
   host.ReloadInstance();
 
-  // Create a RootView which will present a react-native component
-  winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
-  viewOptions.ComponentName(mainComponentName);
-  auto rootView = winrt::Microsoft::ReactNative::CompositionRootView(compositor);
-  rootView.ReactViewHost(winrt::Microsoft::ReactNative::ReactCoreInjection::MakeViewHost(host, viewOptions));
+  // Create a ReactNative Island
+  auto reactNativeIsland =
```

Comment:
Can the compositor and associated code be abstracted into the RN island? Curious to know, not asking for any changes for this PR.